applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -634,7 +640,9 @@ static bool key_value_set(struct items *items, u16_t usage_id, s16_t value)
 
 static void send_report_keyboard(u8_t report_id, struct report_data *rd)
 {
-	__ASSERT_NO_MSG(report_id == REPORT_ID_KEYBOARD_KEYS);
+	__ASSERT_NO_MSG((report_id == REPORT_ID_KEYBOARD_KEYS) ||
+			(report_id == REPORT_ID_BOOT_KEYBOARD));
```

Comment:
We could assert if boot keyboard report is supported too

Reply:
fixed